src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,108 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imdbId: '',
+    imdbUrl: '',
+    imgUrl: '',
+  };
+
+  getMovieData = (event, prop) => {
+    this.setState({
+      [prop]: event.target.value,
+    });
+  }
+
+  submit = (event) => {
+    const { title, description, imdbId, imdbUrl, imgUrl } = this.state;
+
+    this.setState({
+      title: '',
+      description: '',
+      imdbId: '',
+      imdbUrl: '',
+      imgUrl: '',
+    });
+
+    return (
+      this.props.addMovie(
+        event,
+        title,
+        description,
+        imdbId,
+        imdbUrl,
+        imgUrl,
+      )
+    );
+  }
 
   render() {
     return (
-      <form>
-        Put the form here
+      <form
+        className="addMovieForm"
+        onSubmit={this.submit}
+      >
+        <input
+          type="text"
+          value={this.state.title}
+          onChange={event => this.getMovieData(event, 'title')}
+          placeholder="Enter a movie title"
+          className="addMovieForm__element"
+          required
+        />
+
+        <textarea
+          type="text"
+          value={this.state.description}
+          onChange={event => this.getMovieData(event, 'description')}
+          placeholder="Enter a movie description"
+          className="addMovieForm__element"
+          rows="5"
+          required
+        />
+
+        <input
+          type="text"
+          value={this.state.imgUrl}
+          onChange={event => this.getMovieData(event, 'imgUrl')}
```

Comment:
добавь инпуту атрибут name

потом event.target.name
```suggestion
          onChange={this.getMovieData}
```

Reply:
Дякую за пораду
✓